src/h5web/metadata-viewer/EntityTable.tsx

```diff
@@ -47,7 +46,15 @@ function EntityTable(props: Props) {
             <td>{renderShape(entity.shape)}</td>
           </tr>
         )}
-        {isLink(entity) && entity.rawLink && <LinkInfo link={entity.rawLink} />}
+        {entity.link && entity.link.path && (
+          <tr>
+            <th scope="row">{entity.link.class} link</th>
+            <td>
+              {entity.link.file && `${entity.link.file}:`}
+              {entity.link.path}
+            </td>
+          </tr>
+        )}
```

Comment:
If the underlying `link` of the entity has a path (typically if it's soft or external), then we show a row labelled "Soft/External link" containing the `path` to the linked entity (including the `file`, for external links).